django/db/backends/sqlite3/base.py

```diff
@@ -554,19 +554,29 @@ def _sqlite_format_dtdelta(conn, lhs, rhs):
     LHS and RHS can be either:
     - An integer number of microseconds
     - A string representing a datetime
+    - A scalar factor (e.g. float)
     """
-    try:
-        real_lhs = datetime.timedelta(0, 0, lhs) if isinstance(lhs, int) else backend_utils.typecast_timestamp(lhs)
-        real_rhs = datetime.timedelta(0, 0, rhs) if isinstance(rhs, int) else backend_utils.typecast_timestamp(rhs)
-        if conn.strip() == '+':
-            out = real_lhs + real_rhs
+    if conn.strip() in ['+', '-']:
+        try:
+            real_lhs = datetime.timedelta(0, 0, lhs) if isinstance(lhs, int) else backend_utils.typecast_timestamp(lhs)
+            real_rhs = datetime.timedelta(0, 0, rhs) if isinstance(rhs, int) else backend_utils.typecast_timestamp(rhs)
+            if conn.strip() == '+':
+                out = real_lhs + real_rhs
+            else:
+                out = real_lhs - real_rhs
+        except (ValueError, TypeError):
+            return None
+        # typecast_timestamp returns a date or a datetime without timezone.
+        # It will be formatted as "%Y-%m-%d" or "%Y-%m-%d %H:%M:%S[.%f]"
+        return str(out)
+    else:
+        if not isinstance(lhs, (int, float)) or not isinstance(rhs, (int, float)):
```

Comment:
I think instead of `else` you should use
 ```py
elif conn.strip() in ['*', '/']:
 ```
. Since it is including operators other than "*" and "/".